skymarshal/web/templates/device.html

```diff
@@ -0,0 +1,23 @@
+{{ template "header.html" . }}
+
+<div class="theme-panel">
```

Comment:
Should we replace these 2 with a "Something has gone horribly wrong, please file an issue on concourse/concourse" page? It might be better to render this as an error page if the user should never see it.

Reply:
Seems reasonable to me - I suspect this isn't something we'll choose to support, at least not for a while